lib/bankaccounts/checking-account.rb

```diff
@@ -0,0 +1,48 @@
+module BankAccounts
+
+  # CheckingAccount inherits from Account class
+  class CheckingAccount < Account
+    attr_accessor :checks_used
+
+    def initialize(id, initial_balance)
+      super
+      @min_balance = 0
+      check_initial_balance(initial_balance)
```

Comment:
In this case, no, because min_balance is 0 for both CheckingAccount and Account. It would cause it to break if you removed it from SavingsAccount though.